packages/components/src/InputDate/InputDate.tsx

```diff
@@ -38,6 +38,17 @@ interface InputDateProps
    * The minimum selectable date.
    */
   readonly minDate?: Date;
+
+  /**
+   * Whether to show the calendar icon
+   * @default true
+   */
+  readonly showIcon?: boolean;
+
+  /**
+   * Text to display instead of a date value
+   */
+  readonly emptyValuePlaceholder?: string;
```

Comment:
To match the mobile version I think this prop should be updated to be
`emptyValueLabel` (sorry for missing this before).